web/pages/api/contact.js

```diff
@@ -1,6 +1,15 @@
 import { sendSesEmail } from "../../lib/sendSesEmail";
 import { v4 as uuidV4 } from "uuid";
 import dynamoDb from "../../lib/dynamoDb";
+const mailchimpClient = require("@mailchimp/mailchimp_marketing");
+
+mailchimpClient.setConfig({
+  apiKey: process.env.NEXT_PUBLIC_MAILCHIMP_API_KEY,
+  server: process.env.NEXT_PUBLIC_MAILCHIMP_SERVER_PREFIX,
+});
+
+const gatedDocumentListId =
```

Comment:
I think we should store this in Sanity. That way they can use the same component for different purposes

Reply:
Can do. Would this also be something I could use to replace relying on the `organization` variable?